### PR TITLE
Update asteroid-launcher.es.ts

### PR DESCRIPTION
**Pull request by Caballlero**
I do not understand correctly line:23 "Pass Key".  Can you explain what you mean by this?

### CODE REVIEW

**Comment by FlorentRevest** [comment]
https://motorola-global-en-aus.custhelp.com/app/answers/detail/a_id/23847/~/bluetooth-passkey

File: i18n/asteroid-launcher.es.ts [hunk 1]
```diff
@@ -6,92 +6,92 @@
...
         <location filename="../qml/notifications/NotificationView.qml" line="62"/>
         <source>h</source>
-        <translation type="unfinished"></translation>
+        <translation>h</translation>
     </message>
     <message>
         <location filename="../qml/notifications/NotificationView.qml" line="65"/>
         <source>d</source>
-        <translation type="unfinished"></translation>
+        <translation>d</translation>
```
Comment (FlorentRevest) [inline i18n/asteroid-launcher.es.ts]: Those letters stand for minute, hour and day in case you were wondering

**Comment by Caballlero** [comment]
I understand "PassKey" but you have wrote "Pass Key"  with one space, it is a little diferent.

**Comment by FlorentRevest** [comment]
It's the same

**Comment by Caballlero** [comment]
OK 
Thank you for clear explanation & Thank you for create AsteroidOS: I am ok on my translation. Ready to Merge.  :  )

**Pull request merged**